nuget/lib/dependabot/nuget/metadata_finder.rb

```diff
@@ -144,11 +145,12 @@ def auth_header
                            .find { |r| r.fetch(:source) }&.fetch(:source)
         url = source&.fetch(:url, nil) || source&.fetch("url")
 
-        token = credentials
-                .select { |cred| cred["type"] == "nuget_feed" }
-                .find { |cred| cred["url"] == url }
-                &.fetch("token", nil)
+        matching_credentials = credentials
```

Comment:
nit: name this `matching_credential` because I was expecting an array.

Reply:
Done.